presentation/src/main/java/com/example/presentation/ui/components/ProfileImage.kt

```diff
@@ -0,0 +1,45 @@
+package com.example.presentation.ui.components
+
+import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Brush
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import com.bumptech.glide.integration.compose.ExperimentalGlideComposeApi
+import com.bumptech.glide.integration.compose.GlideImage
+import com.example.presentation.ui.theme.BasePink
+import com.example.presentation.ui.theme.BasePurple
+
+@OptIn(ExperimentalGlideComposeApi::class)
+@Composable
+fun GradientProfileImage(
+    modifier: Modifier = Modifier,
+    userName: String,
+    imageUrl: String,
+    size: Dp,
+    borderWidth: Dp = 2.dp,
+    startColor: Color = BasePurple,
+    endColor: Color = BasePink,
```

Comment:
kotlin 매개변수 우선 순위에 따르면 아래처럼 바꿔주면 좋을 것 같아요!
```suggestion
    userName: String,
    imageUrl: String,
    size: Dp,
    modifier: Modifier = Modifier,  <-- 디폴트 값을 가진 매개변수 중에 제일 최상위
    borderWidth: Dp = 2.dp,
    startColor: Color = BasePurple,
    endColor: Color = BasePink,
```